src/components/reusable_components/InvestItemList.js

```diff
@@ -16,27 +29,183 @@ export default function InvestItemList(props) {
         </View>
       ) : (
         <View style={styles.listItems}>
-          {listData.map((item) => {
-            return (
-              <Text key={item.id} style={styles.text}>
-                {item.name}
+          {listData.length === 0 ? (
+            <View style={styles.noResultsContainer}>
+              <Text style={styles.text}>
+                You don't have any investors yet. Press the plus icon above to
```

Comment:
Is there supposed to be a plus icon somewhere ?

Reply:
That will be added when I get to the create investor ticket